assets/chatglm.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- type: model
+  name: ChatGLM
+  organization: ChatGLM
+  description: ChatGLM is a Chinese-English language model with question and answer
+    and dialogue functions, and is aimed at a Chinese audience.
+  created_date: 2023-03-14
+  url: https://chatglm.cn/blog
+  model_card: ''
+  modality: text
+  analysis: Performance evaluated on English and Chinese language benchmark tests.
+  size: 6B parameters
+  dependencies: [GLM-130B]
```

Comment:
I think something is wrong here, since to go from GLM to ChatGLM:

1. Presumably the ChatGLM model should be the same or bigger than the GLM model
2. Presumably there should be an additional (chat-related) dataset that you take GLM and future train on to get ChatGLM.